### PR TITLE
JAMES-1890 reuse JMS connection as it's a thread-safe object

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by mbaechler** [comment]
It breaks tests in eclipse (cassandra jmap integration testing leaks fd)

**Comment by mbaechler** [comment]
ok, it was actually broken before this change

File: server/queue/queue-jms/src/main/java/org/apache/james/queue/jms/JMSMailQueue.java [hunk 1]
```diff
@@ -69,17 +72,22 @@
...
+    protected final Connection connection;
     protected final MailQueueItemDecoratorFactory mailQueueItemDecoratorFactory;
     protected final Metric enqueuedMailsMetric;
     protected final Logger logger;
     public final static String FORCE_DELIVERY = "FORCE_DELIVERY";
 
     public JMSMailQueue(ConnectionFactory connectionFactory, MailQueueItemDecoratorFactory mailQueueItemDecoratorFactory, String queueName, Metric enqueuedMailsMetric, Logger logger) {
-        this.connectionFactory = connectionFactory;
+        try {
+        	connection = connectionFactory.createConnection();
```
Comment (chibenwa) [inline server/queue/queue-jms/src/main/java/org/apache/james/queue/jms/JMSMailQueue.java]: Indent ?

Comment (mbaechler) [inline server/queue/queue-jms/src/main/java/org/apache/james/queue/jms/JMSMailQueue.java]: fixed

**Pull request merged**